example/game/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,35 @@
+<resources>
+    <string name="app_name">Game</string>
+
+    <string name="start_game_description">Please select one option</string>
+    <string name="start_game">Start Game</string>
+    <string name="join_game">Join Game</string>
+    <string name="play">Play Game</string>
+
+    <string name="client">Client Side</string>
```

Comment:
Where do you need that?

Reply:
This does not sound like something encouraging to play :)
If that goes to an AppBar, how about printing a category name, or Good Luck, etc. I don't know where do you use it.